src/ert/run_models/everest_run_model.py

```diff
@@ -211,6 +212,11 @@ def create(
         )
         ert_templates = read_templates(config_dict)
 
+        for datafile, data in get_internal_files(everest_config).items():
+            datafile.parent.mkdir(exist_ok=True, parents=True)
+            with open(datafile, "w", encoding="utf-8") as fp:
```

Comment:
`datafile` is a `pathlib.Path`object, then you can do `datafile.write_text(..)` directly.